lib/need_for_type/file_manager.rb

```diff
@@ -2,30 +2,16 @@ module NeedForType
   class FileManager
     attr_accessor :content
 
-    def initialize(difficulty)
-      @path = "#{NeedForType::GEM_ROOT}/assets/#{difficulty_to_s(difficulty)}/"
+    def initialize()
+      @path = "#{NeedForType::GEM_ROOT}/assets/texts/"
     end
 
     # Reads random file from difficulty folder
     def get_random_text
-      directory = Dir[@path]
-      random_file = rand(1..directory.count)
+      directory_count = Dir[File.join(@path, '**', '*')].count { |file| File.file?(file) }
```

Comment:
it was used to verify if the file was actually a file or not but this is not needed for now so I will leave it at the count ;)